main.py

```diff
@@ -301,6 +303,29 @@ async def user_stats(interaction: discord.Interaction, member: discord.Member):
 async def mini(interaction: discord.Interaction):
   """What does this even do??"""
   await interaction.response.send_message("This command was developed by mini")
+  
+@bot.tree.command(name="jerome")
+async def jerome(interaction: discord.Interaction):
+  r = requests.get('https://api.quotable.io/random')
```

Comment:
We should not use the requests library as it is blocking, if anyone runs this command, all other users have to wait until this command finishes running before their commands can be run. This would also affect mushroom farming as well.
Refer to [this](https://discordpy.readthedocs.io/en/stable/faq.html#what-does-blocking-mean) for more info and how to solve it